app/assets/javascripts/float-kit/addon/components/d-default-toast.gjs

```diff
@@ -12,6 +29,13 @@ const DDefaultToast = <template>
     }}
     ...attributes
   >
+    {{#if @data.progressBar}}
+      <div
+        class="fk-d-default-toast__progress-bar"
+        {{(modifier ToastProgress duration=@duration)}}
```

Comment:
I don't think there's any need for the `modifier` helper here. We should just be able to use it directly 👀 
```suggestion
        {{ToastProgress duration=@duration}}
```